modules/rpc/handler.go

```diff
@@ -34,6 +35,8 @@ type contextInboundMiddleware struct {
 }
 
 func (f contextInboundMiddleware) Handle(ctx context.Context, req *transport.Request, resw transport.ResponseWriter, handler transport.UnaryHandler) error {
+	stopwatch := stats.RPCHandleTimer.Timer("rpc.handler." + req.Procedure + ".time").Start()
```

Comment:
Prometheus backend is using "_" as separators..
https://prometheus.io/docs/practices/naming/